### PR TITLE
Adding Bio and Socials

### LINKED ISSUE #6: Add bio section
From pod-3.1.2-portfolio created by [RashikaKarki](https://github.com/RashikaKarki): MLH-Fellowship/pod-3.1.2-portfolio#6  

It would be great to have a bio section in the portfolio, where you all can talk more about who you are, and your programming story.

### PR DESCRIPTION
**Pull request by harshgeek4coder**
## Bio and Social Links

- Adding Bio in About Me Section
- Adding Social Links in Footer Section

Closes #6 #9

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -10,4 +10,6 @@ github_username:  mlh-fellowship
 sass:
   sass_dir: _sass
 
-icon: /assets/img/icon.jpg
\ No newline at end of file
+icon: /assets/img/icon.jpg
+
```
Comment (Rishit-dagli) [inline _config.yml:14]: extra lines

File: Gemfile [hunk 2]
```diff
@@ -10,3 +10,5 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
 
 gem "webrick", "~> 1.7"
+
```
Comment (Rishit-dagli) [inline Gemfile:13]: extra lines

File: _data/about.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- info: The MLH Fellowship is a remote internship alternative for aspiring technologists. Spend 12 weeks building your skills by
+        collaborating on real-world projects.Founded in 2013, Major League Hacking (MLH) is the largest community of early career developers 
+        worldwide.We support 115,000+ student developers, designers, and engineers each year through thousands of on-campus events like 
+        workshops, hackathons, and career fests. MLH has been a community first, mission driven organization from the beginning.
+
```
Comment (Rishit-dagli) [inline _data/about.yml:5]: extra lines

File: _data/education.yml [hunk 4]
```diff
@@ -7,4 +7,5 @@
   institute: MLH Fellowship
   dates: Summer 2021
   logo: fellowship.svg
+
```
Comment (Rishit-dagli) [inline _data/education.yml:10]: Why is this file changed?

File: _data/socials.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+- icon: Web
+  icon_pack: fas
+  class: fab fa-firefox
+  link: 'https://fellowship.mlh.io/'
+
+- icon: Twitter
+  icon_pack: fab
+  class: fab fa-twitter
+  link: 'https://twitter.com/MLHacks?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor'
```
Comment (Rishit-dagli) [inline _data/socials.yml:9]: Let's reference this from `_config.yml`?

File: _data/socials.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  
+- icon: Github  # Alternatively, use `google-scholar` icon from `ai` icon pack
+  icon_pack: fas
+  class: fab fa-github
+  link: 'https://github.com/MLH-Fellowship'
```
Comment (Rishit-dagli) [inline _data/socials.yml:14]: Let's reference this too from `_config.yml`?

File: _data/socials.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+- icon: Linkedin
+  icon_pack: fab
+  class: fab fa-linkedin
+  link: 'https://www.linkedin.com/school/mlh-fellowship/'
```
Comment (Rishit-dagli) [inline _data/socials.yml:19]: Sam here. Let's reference this too from `_config.yml`?

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]

File: _data/education.yml (hunk 4 above)
Comment (harshgeek4coder) [reply _data/education.yml:10]: Had some custom tweaks, removed extra lines though

File: _data/socials.yml (hunk 5 above)
Comment (harshgeek4coder) [reply _data/socials.yml:9]: we can, but i followed same convention -> YML files and HTML templates to remain same, if required in future to add extra elements

File: _data/about.yml (hunk 3 above)
Comment (harshgeek4coder) [reply _data/about.yml:5]: Done

File: Gemfile (hunk 2 above)
Comment (harshgeek4coder) [reply Gemfile:13]: Done

File: _config.yml (hunk 1 above)
Comment (harshgeek4coder) [reply _config.yml:14]: Done

**Comment by harshgeek4coder** [comment]
committed the requested changes

File: assets/css/main.scss [hunk 8]
```diff
@@ -2,3 +2,7 @@
 ---
 
 @import 'all';
+
+
+
+
```
Comment (shreyaparadkar) [inline assets/css/main.scss:8]: extra lines

**Review by shreyaparadkar — CHANGES_REQUESTED** [review]

**Review by Rishit-dagli — APPROVED** [review]
Thanks @harshgeek4coder 👍

**Review by shreyaparadkar — APPROVED** [review]

**Comment by ayushmaang** [comment]
LGTM. Like the fact that everyone approves beforehand.

**Pull request merged**